app/src/main/java/org/oppia/app/player/state/answerhandling/InteractionAnswerHandler.kt

```diff
@@ -14,6 +14,16 @@ interface InteractionAnswerHandler {
    */
   fun isExplicitAnswerSubmissionRequired(): Boolean = true
 
+  /** Return the current answer error messages. */
```

Comment:
Please indicate in this method's KDoc that the returned value is null when there is no error. I think that's an important detail in the API being described here to help calling code use it correctly.

Reply:
Done